File: 4_single_level_inheritance.cpp
#include <bits/stdc++.h>
#define ll long long int
using namespace std;
const int N = 1e7 + 1;
class Car
{
public:
    string name;
    void display()
    {
        cout << name << " is a car";
    }
};

class Toyota : public Car
{
public:
    Toyota(string name)
    {
        this->name = name;
    }
};

int main()
{
    Toyota obj("Toyota");
    obj.display();
}

File: 4_single_level_inheritance.py
class Car:
    def __init__(self, name) -> None:
        self.name = name

    def display(self):
        print(self.name, 'is a car')


class Toyota(Car):
    def __init__(self, name) -> None:
        super().__init__(name)


obj = Toyota("Toyota")

obj.display()